.github/workflows/dapr-base-containers.yaml

```diff
@@ -14,17 +14,25 @@
 name: dapr-base-containers
 
 on:
+  # Manual trigger
+  workflow_dispatch:
```

Comment:
Also added the ability for a maintainer to trigger the workflow manually. After the PR is merged, it will need to be triggered.